### PR TITLE
[WIP] Rolfes third branch

### PR DESCRIPTION
**Pull request by rolfedh**
Updated commands to send commands to the Swarm manager instead of to Docker Engine host where the Swarm manager resides.

Signed-off by Rolfe Dlugy-Hegwer <rollfedh-contractor@docker.com>

### CODE REVIEW

**Comment by rolfedh** [comment]
THIS PR IS FOR WIP REVIEW

**Comment by nishanttotla** [comment]
Since this doc is under review, can we change to using standard ports for Swarm (3375 for non-TLS instead of 2360 or others) so that we don't confuse people?

cc @dongluochen @moxiegirl

File: docs/install-w-machine.md [hunk 1]
```diff
@@ -101,64 +101,113 @@ Here, you connect to each of the hosts and create a Swarm manager or node.
 
         $ docker-machine ls
         NAME      ACTIVE   DRIVER       STATE     URL                         SWARM   DOCKER   ERRORS
-        agent1    -        virtualbox   Running   tcp://192.168.99.102:2376           v1.9.1   
-        agent2    -        virtualbox   Running   tcp://192.168.99.103:2376           v1.9.1   
-        manager   *        virtualbox   Running   tcp://192.168.99.100:2376           v1.9.1   
+        node1_host    -        virtualbox   Running   tcp://192.168.99.102:2376           v1.10.0
```
Comment (vieux) [inline docs/install-w-machine.md]: spacing is wrong here

**Comment by rolfedh** [comment]
Thanks for reviewing, Nishant! I'll do that.

**Comment by rolfedh** [comment]
I fixed the spacing issues and changed the mapped port number to 3375.

I could really use your help fixing the TLS authentication issues that are
causing the Swarm nodes to show up in the Swarm master's info with "Error:
Cannot connect to the docker engine endpoint".

File: docs/install-w-machine.md [hunk 2]
```diff
@@ -101,64 +94,114 @@ Here, you connect to each of the hosts and create a Swarm manager or node.
...
+        Strategy: spread
+        Filters: health, port, dependency, affinity, constraint
+        Nodes: 2
+         (unknown): 192.168.99.102:3375
+          └ Status: Pending
+          └ Containers: 0
+          └ Reserved CPUs: 0 / 0
+          └ Reserved Memory: 0 B / 0 B
+          └ Labels:
+          └ Error: Cannot connect to the docker engine endpoint
```
Comment (nishanttotla) [inline docs/install-w-machine.md]: Both your nodes here are unreachable. Is this intentional?

**Comment by nishanttotla** [comment]
@rolfedh we fixed some immediate issues with some existing docs, so your branch now has merge conflicts which you need to resolve. I would recommend dropping changes that conflict with #1778.

cc @dongluochen @moxiegirl

**Comment by rolfedh** [comment]
@nishant @dongluochen @moxiegirl Thank you for responding to/resolving the
issue I raised about the nodes. I'll manually merge my changes into the
updated file.

Rolfe

**Comment by moxiegirl** [comment]
test please

**Comment by rolfedh** [comment]
test this please

**Comment by rolfedh** [comment]
test this please

**Comment by rolfedh** [comment]
test this please

**Comment by rolfedh** [comment]
test this please

**Comment by rolfedh** [comment]
test this please

**Pull request closed without merging**